### DISCUSSION TITLE
Show how many added sorces at bottom of sources page

### DISCUSSION THREAD

**Discussion by worldlydev** [wstyres/Zebra#814] [Ideas]
![A4BF27C7-8DF7-485A-ACDB-9FF7F821CD23](https://user-images.githubusercontent.com/51668635/72212539-5021d500-34de-11ea-8b47-f752ebe622c7.jpeg)

![E7E1C232-B3B9-4E99-AC5E-D8F536E7B541](https://user-images.githubusercontent.com/51668635/72212538-5021d500-34de-11ea-8cbf-c287d5f2de4b.jpeg)
![9676831B-98C7-4174-AE4D-6CD06A44C726](https://user-images.githubusercontent.com/51668635/72212540-5021d500-34de-11ea-9459-14135022225a.jpeg)

**Comment by wstyres** [comment]
We'll see how this fits into the final design but I don't see that much purpose to adding it

**Comment by worldlydev** [comment]
> We'll see how this fits into the final design but I don't see that much purpose to adding it

Not really on my wishlist but I could see why it would be useful. I just open issues on GitHub so that other peoples suggestions don't get lost in the discord.

**Comment by worldlydev** [comment]
![88A9C83D-D6E5-42B3-B565-177BAA9507CE](https://user-images.githubusercontent.com/51668635/79687152-8a211680-8245-11ea-82ff-769633c8927b.png)
